diff --git a/tools/xorwow_precomputed_generator.cpp b/tools/xorwow_precomputed_generator.cpp
index 1e9dad0..be66f53 100644
--- a/tools/xorwow_precomputed_generator.cpp
+++ b/tools/xorwow_precomputed_generator.cpp
@@ -27,2 +27,2 @@
-  const int XORWOW_N = 5;  // 5 values
-  const int XORWOW_M = 32; // 32-bit each
+const int XORWOW_N = 5; // 5 values
+const int XORWOW_M = 32; // 32-bit each
@@ -33 +33 @@ const int XORWOW_JUMP_MATRICES = 32;
-const int XORWOW_JUMP_LOG2 = 2;
+const int XORWOW_JUMP_LOG2     = 2;
@@ -50,5 +50,5 @@ struct rocrand_xorwow_state
-        x[0] = x[1];
-        x[1] = x[2];
-        x[2] = x[3];
-        x[3] = x[4];
-        x[4] = (x[4] ^ (x[4] << 4)) ^ (t ^ (t << 1));
+        x[0]                 = x[1];
+        x[1]                 = x[2];
+        x[2]                 = x[3];
+        x[3]                 = x[4];
+        x[4]                 = (x[4] ^ (x[4] << 4)) ^ (t ^ (t << 1));
@@ -64 +64 @@ void generate_matrices()
-    for (int i = 0; i < XORWOW_N; i++)
+    for(int i = 0; i < XORWOW_N; i++)
@@ -66 +66 @@ void generate_matrices()
-        for (int j = 0; j < XORWOW_M; j++)
+        for(int j = 0; j < XORWOW_M; j++)
@@ -69,2 +69,2 @@ void generate_matrices()
-            const unsigned int b = 1U << j;
-            for (int k = 0; k < XORWOW_N; k++)
+            const unsigned int   b = 1U << j;
+            for(int k = 0; k < XORWOW_N; k++)
@@ -78 +78 @@ void generate_matrices()
-            for (int k = 0; k < XORWOW_N; k++)
+            for(int k = 0; k < XORWOW_N; k++)
@@ -91 +91 @@ void generate_matrices()
-        for (int k = 1; k < XORWOW_JUMP_MATRICES; k++)
+        for(int k = 1; k < XORWOW_JUMP_MATRICES; k++)
@@ -113 +113 @@ void generate_matrices()
-        for (int k = 1; k < XORWOW_JUMP_MATRICES; k++)
+        for(int k = 1; k < XORWOW_JUMP_MATRICES; k++)
@@ -122,2 +122,3 @@ void generate_matrices()
-int main(int argc, char const *argv[]) {
-    if (argc != 2 || std::string(argv[1]) == "--help")
+int main(int argc, char const* argv[])
+{
+    if(argc != 2 || std::string(argv[1]) == "--help")
@@ -126 +127,3 @@ int main(int argc, char const *argv[]) {
-        std::cout << "  ./xorwow_precomputed_generator ../../library/include/rocrand/rocrand_xorwow_precomputed.h" << std::endl;
+        std::cout << "  ./xorwow_precomputed_generator "
+                     "../../library/include/rocrand/rocrand_xorwow_precomputed.h"
+                  << std::endl;
@@ -133 +136 @@ int main(int argc, char const *argv[]) {
-    std::ofstream fout(file_path, std::ios_base::out | std::ios_base::trunc);
+    std::ofstream     fout(file_path, std::ios_base::out | std::ios_base::trunc);
